src/interchange/src/avro.rs

```diff
@@ -1263,7 +1263,7 @@ impl DebeziumDeduplicationState {
                     };
                     let mut row_iter = match update.after.as_ref() {
                         None => {
-                            error!("Snapshot row at pos {:?}, time {:?} was unexpectedly not an insert.", coord, upstream_time_millis);
+                            error!("Snapshot row at pos {:?}, time {:?} in {} was unexpectedly not an insert.", coord, upstream_time_millis, debug_name);
```

Comment:
could you pass the upstream_time_millis through the timestamp formatter so that it's not just an int, and make the `time` -> `message_time` so that it's more obviously the same kind of thing as the other places we log this:

```suggestion
                            error!(
                                "Snapshot row at pos {:?}, message_time {} in {} was unexpectedly not an insert.",
                                coord, fmt_timestamp(upstream_time_millis), debug_name);
```